content/docs/1.4.0/volumes-and-nodes/multidisk.md

```diff
@@ -44,7 +44,7 @@ Once those two conditions are met, you should be allowed to remove the disk.
 There are two global settings affect the scheduling of the volume.
 
 - `StorageOverProvisioningPercentage` defines the upper bound of `ScheduledStorage / (MaximumStorage - ReservedStorage)`. The default value is `200` (%). That means we can schedule a total of 300 GiB Longhorn volumes on a 200 GiB disk with 50G reserved for the root file system. Because normally people won't use that large amount of data in the volume, and we store the volumes as sparse files.
-- `StorageMinimalAvailablePercentage` defines when a disk cannot be scheduled with more volumes. The default value is `10` (%). The bigger value between `MaximumStorage * StorageMinimalAvailablePercentage / 100` and `MaximumStorage - ReservedStorage` will be used to determine if a disk is running low and cannot be scheduled with more volumes.
+- `StorageMinimalAvailablePercentage` defines when a disk cannot be scheduled with more volumes. The default value is `25` (%). The bigger value between `MaximumStorage * StorageMinimalAvailablePercentage / 100` and `MaximumStorage - ReservedStorage` will be used to determine if a disk is running low and cannot be scheduled with more volumes.
```

Comment:
According to this KB which is derived directly from the code https://www.suse.com/support/kb/doc/?id=000021268, we don't seem to compare `MaximumStorage * StorageMinimalAvailablePercentage / 100` and `MaximumStorage - ReservedStorage`

Reply:
Quite right.  I should have caught that.  I remember the thread with that KB being reviewed.

Reply:
Good catch! And a good opportunity to demystify scheduling by correctly documenting how it works.

Reply:
Rewritten.  See if that's better.